packages/orbit-components/src/tmp-tailwind/Textarea/index.tsx

```diff
@@ -0,0 +1,146 @@
+  const {
+    size = SIZE_OPTIONS.NORMAL,
+    disabled,
+    resize = RESIZE_OPTIONS.VERTICAL,
+    dataTest,
+    id,
+    spaceAfter,
+    fullHeight,
+    defaultValue,
+    value,
+    label,
+    name,
+    error,
+    placeholder,
+    helpClosable = true,
+    maxLength,
+    onChange,
+    onFocus,
+    onBlur,
+    tabIndex,
+    help,
+    rows,
+    readOnly,
+    required,
+    dataAttrs,
+  }: Props = props;
+
+  const forID = useRandomId();
+  const inputId = id || forID;
+
+  const hasTooltip = Boolean(error || help);
+
+  const {
+    tooltipShown,
+    tooltipShownHover,
+    setTooltipShown,
+    setTooltipShownHover,
+    labelRef,
+    iconRef,
+    handleFocus,
+  } = useErrorTooltip({ onFocus, hasTooltip });
+
+  const shown = tooltipShown || tooltipShownHover;
+
+  return (
+    <label
+      className={cx(
+        "orbit-textarea",
+        "font-base relative flex w-full flex-col",
+        fullHeight && "h-full flex-1",
+        spaceAfter && getSpaceAfterClasses(spaceAfter),
+      )}
+      ref={labelRef}
+    >
+      {label && (
+        <FormLabel
+          filled={!!value}
+          error={!!error}
+          help={!!help}
+          required={required}
+          disabled={disabled}
+          iconRef={iconRef}
+          onMouseEnter={() => setTooltipShownHover(true)}
+          onMouseLeave={() => setTooltipShownHover(false)}
+        >
+          {label}
+        </FormLabel>
+      )}
+
+      <textarea
+        className={cx(
+          "w-full appearance-none",
+          "font-base",
+          "rounded-normal relative box-border block overflow-auto",
+          "min-h-form-box-normal",
+          "duration-fast transition-shadow ease-in-out",
+          "border border-transparent",
+          size === SIZE_OPTIONS.SMALL && "leading-small text-small px-sm py-xs",
+          size === SIZE_OPTIONS.NORMAL && "text-normal p-sm leading-normal",
+          error
+            ? "shadow-form-element-error hover:shadow-form-element-error"
+            : "shadow-form-element",
+          disabled ? "cursor-not-allowed" : "cursor-text",
+          fullHeight && "h-full flex-1",
+          disabled || readOnly
+            ? "bg-form-element-disabled-background"
+            : "bg-form-element-background",
+          "[&::placeholder]:form-element-placeholder",
+          !(disabled || readOnly) && "hover:shadow-form-element-hover",
+        )}
+        data-state={getFieldDataState(!!error)}
+        data-test={dataTest}
+        aria-required={!!required}
+        id={inputId}
+        name={name}
+        value={value}
+        defaultValue={defaultValue}
+        disabled={disabled}
+        placeholder={String(placeholder)}
+        maxLength={maxLength}
+        onChange={onChange}
+        rows={rows}
+        onFocus={handleFocus}
+        onBlur={onBlur}
+        tabIndex={tabIndex ? Number(tabIndex) : undefined}
+        readOnly={readOnly}
+        ref={ref}
+        style={{ resize }}
```

Comment:
I think we can avoid this and use tailwind classes

`resize === RESIZE_OPTIONS.VERTICAL ? "resize-y" : "resize-none"`